### ISSUE TITLE
Read aloud with screen reader should stay in sync

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/reading-system-testing#18]
The current read aloud test book does not address screen reader issues. We should add two tests:

1. The location of focus for the screen reader should be where read aloud playback starts.
2. When read aloud stops, it should be possible for the screen reader to be at the same location.

If these tests are added, it would be excellent to assign this to environments that use a screen reader.

The one test, "ReadAloud-610 Text is emphasised as it is spoken by read aloud" is the only one that cannot be performed by a blind screen  reader user. If we can mark that one as not applicable (NA) and if that is not counted in the percentage, then this would work OK.
